### PR TITLE
[Feat] [snt9|2_5] Added a case: Xception-based Achieve leaf recognition

### PR DESCRIPTION
**Pull request by ilovexsir**
## Test environment
Hardware: 1*ascend-snt9|ARM: 24核 96GB
Image: python3.9-ms2.5.0-cann8.0.0.beta1

## Test Report
Accuracy: 19.2%
Inference Time: 112 ms per image
Training Time: 61 minutes
Total Script Runtime: 66 minutes

## Related Issues

Fixes #4

### CODE REVIEW

File: dev/ascend/snt9/2_5/Xception实现植物叶子识别.ipynb [hunk 1]
```diff
@@ -0,0 +1,707 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "13fb5a39-4834-421d-96c9-fd8b57e6f6c2",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "batch_size = 256  # 批量大小\n",
```
Comment (hongziqi) [inline dev/ascend/snt9/2_5/Xception实现植物叶子识别.ipynb:53]: Try reducing the `batch_size` and train again

File: dev/ascend/snt9/2_5/Xception实现植物叶子识别.ipynb [hunk 2]
```diff
@@ -0,0 +1,707 @@
...
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "e5bf4f6b-29b4-40a1-a23c-c12299cce6e3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# 设置学习率\n",
+    "num_epochs = 5\n",
```
Comment (hongziqi) [inline dev/ascend/snt9/2_5/Xception实现植物叶子识别.ipynb:473]: After reducing `batch_size`, increase the `num_epochs`

**Comment by hongziqi** [comment]
> ## PR Type
> > Please uncomment only one of the following lines, remove the leading spaces, and leave the other options commented out:
> > /kind cpu-case
> 
> ## Description of Changes
> ## Test environment
> Hardware: 1*ascend-snt9|ARM: 24核 96GB Image: python3.9-ms2.5.0-cann8.0.0.beta1
> 
> ## Test Report
> Accuracy: 19.2% Inference Time: 112 ms per image Training Time: 61 minutes Total Script Runtime: 66 minutes
> 
> ## Related Issues
> Fixes #4
> 
> ## Additional Notes

Please update test report after re-running script, this case is a cpu-case? if not, please update `PR Type`.

**Comment by ilovexsir** [comment]
I always update it, please review it.

File: dev/ascend/snt9/2_5/Xception实现植物叶子识别.ipynb [hunk 3]
```diff
@@ -0,0 +1,707 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d8f1c8d4-828a-4673-8650-784066588314",
+   "metadata": {},
+   "source": [
+    "### 数据集下载及数据处理\n",
+    "\n",
+    "通过download库下载魔乐平台的数据集。"
```
[the remark this answers is not in the document]
Comment (hongziqi) [reply dev/ascend/snt9/2_5/Xception实现植物叶子识别.ipynb:96]: Please update the description here

**Comment by hongziqi** [comment]
> ## PR Type
> > Please uncomment only one of the following lines, remove the leading spaces, and leave the other options commented out:
> > /kind Ascend-case
> 
> ## Description of Changes
> ## Test environment
> Hardware: 1*ascend-snt9|ARM: 24核 96GB Image: python3.9-ms2.5.0-cann8.0.0.beta1
> 
> ## Test Report
> Accuracy: 100% Inference Time: 10.2 ms per image Training Time: 49 minutes Total Script Runtime: 55 minutes
> 
> ## Related Issues
> Fixes #4
> 
> ## Additional Notes

Training Time: 49 minutes  maybe not correct, you can use `%%time` in the cell top, and check the `wall time`

**Comment by ilovexsir** [comment]
I update it.

**Comment by hongziqi** [comment]
`通过download库下载魔乐平台的数据集。` haven't modified it yet. Please check the comment above.

**Comment by ilovexsir** [comment]
> `通过download库下载魔乐平台的数据集。` haven't modified it yet. Please check the comment above.

done

**Comment by hongziqi** [comment]
> ## PR Type
> > Please uncomment only one of the following lines, remove the leading spaces, and leave the other options commented out:
> > /kind Ascend-case
> 
> ## Description of Changes
> ## Test environment
> Hardware: 1*ascend-snt9|ARM: 24核 96GB Image: python3.9-ms2.5.0-cann8.0.0.beta1
> 
> ## Test Report
> Accuracy: 100% Inference Time: 0.694 ms per image Training Time: 22 minutes Total Script Runtime: 30 minutes
> 
> ## Related Issues
> Fixes #4
> 
> ## Additional Notes

Please update you training time and script runtime, only use 4mins in trainning cell.
image

**Comment by ilovexsir** [comment]
ok, I update it.

**Comment by hongziqi** [comment]
> ok, I update it.

Please also update `Total Script Runtime`

**Review by hongziqi — APPROVED** [review]
